packages/perseus/src/renderer.tsx

```diff
@@ -1289,12 +1289,6 @@ class Renderer extends React.Component<Props, State> {
             );
         }
         if (node.type === "math") {
-            // Replace uses of \begin{align}...\end{align} which KaTeX doesn't
-            // support (yet) with \begin{aligned}...\end{aligned} which renders
-            // the same is supported by KaTeX.  It does the same for align*.
-            // TODO(kevinb) update content to use aligned instead of align.
-            const tex = node.content.replace(/\{align[*]?\}/g, "{aligned}");
```

Comment:
MathJax supports the `\begin{align}` macro just fine (tested in Storybook).

Reply:
Removing this code might break the exercise editor, which still uses KaTeX behind the scenes to validate that the input TeX _can_ render with KaTeX. This gets complicated to reason about, because the reason it does that is to ensure that the TeX content creators type is compatible with old versions of the mobile app, which use KaTeX (and old versions of Perseus which have this replacement logic, so `\begin{align}` will work fine there).

I _think_ we could move this logic to the exercise editor and it would be fine. But I'd lean toward keeping this code the same until we can remove KaTeX everywhere. If you agree, can you add a TODO comment here referencing [LEMS-1608](https://khanacademy.atlassian.net/browse/LEMS-1608)?

Reply:
FWIW, I looked up our new [mobile deprecation policy](https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/2883780614/ADR+732+Formalize+the+graphql+deprecation+policy+for+fields+used+by+the+mobile+app) and I believe we will be able to completely remove KaTeX starting September 18th!

Reply:
I'd like to move forward and land this, so I'll do as Ben suggests and restore this code with a new TODO.  Thanks for thinking of that Ben!